docs/modules/configuration/pages/secrets-management.adoc

```diff
@@ -169,3 +169,10 @@ image::vault.png[Secrets in Vault,width=70%]
 db.connection.test.username=HASHI_CORP_VAULT(secret/vividus/test/username)
 db.connection.test.password=HASHI_CORP_VAULT(secret/vividus/test/password)
 ----
+
+=== xref:plugins:plugin-azure-resource-manager.adoc#_secrets_management_tools[Azure Key Vault]
+==== Configuration
+
+Supported getting property values from Azure Key Vault.
+Please see xref:plugins:plugin-azure-resource-manager.adoc#_configuration[Configuration section] to get more
+information on how configure required plugin.
```

Comment:
```suggestion
Check the xref:plugins:plugin-azure-resource-manager.adoc#_configuration[corresponding section] in Azure Resource Manager Plugin.
```

also make sure link points to `Secrets Management Tools` section

Reply:
added link on `_secrets_management_tools` in plugin-azure-resource-manager.adoc